health/health.go

```diff
@@ -131,10 +144,12 @@ func NewTracker(bus *eventbus.Bus) *Tracker {
 	}
 
 	cli := bus.Client("health.Tracker")
-	return &Tracker{
+	t := &Tracker{
 		eventClient: cli,
 		changePub:   eventbus.Publish[Change](cli),
 	}
+	t.timer = t.clock().AfterFunc(time.Minute, t.timerSelfCheck)
```

Comment:
I got excited that we might be able to get rid of this delicate timer handling, but I guess we still reset it on the self check. I wonder if maybe as a separate cleanup, we could change it to use a long-running timer, now that it no longer needs to coordinate with the scheduling mechanism? (Not for this change, though)